packages/tasit-account/src/ethers.test.js

```diff
@@ -16,7 +16,7 @@ const waitForEvent = async (eventName, expected) => {
   });
 };
 
-const ropstenProvider = ethers.getDefaultProvider("ropsten");
+const provider = new ethers.providers.JsonRpcProvider();
```

Comment:
Where do we give it the ganache config?

Also, many of the account-related tests probably don't even need a provider. Just the ones where we're checking the balance. I wonder if for clarity we should only define the provider farther down in the test suite in a describe block for provider-related tasit-account tests

Reply:
For default, ethers JsonRpcProvider uses 127.0.0.1:8545 (same as ganache-cli).
Ok, I'll instantiate provider only where it's needed.

Reply:
Oh cool, good to know about the defaults.